app/views/ReportView/components/Summary/index.tsx

```diff
@@ -11,36 +11,43 @@ const Summary = ({
   templateName,
   ...props
 }: SummaryProps): JSX.Element => {
-  if (templateName === 'probe') {
+  // TODO remove backup template name checks when data is updateed in prod
+  const summarySection = props.visibleSections.find(element => ['summary-genomic', 'summary-tgr', 'summary-pcp', 'summary-probe'].includes(element));
+  if (summarySection === 'summary-probe' || templateName === 'probe') {
     return (
       <ProbeSummary {...props} />
     );
   }
 
-  if (templateName === 'pharmacogenomic') {
+  if (summarySection === 'summary-pcp' || templateName === 'pharmacogenomic') {
+    console.log(templateName);
```

Comment:
Are we keeping the console logs on purpose?

Reply:
ah, no